subsys/bluetooth/audio/ascs.c

```diff
@@ -1378,6 +1378,68 @@ static int ase_config(struct bt_ascs *ascs, struct bt_ascs_ase *ase,
 	return 0;
 }
 
+int bt_audio_unicast_server_config_ase(struct bt_conn *conn,
+		struct bt_audio_stream *stream, struct bt_codec *codec,
+		const struct bt_codec_qos_pref *qos_pref)
+{
+	struct bt_ascs *ascs;
+	struct bt_ascs_ase *ase;
+	struct bt_audio_ep *ep;
+
+	CHECKIF(conn == NULL || stream == NULL || codec == NULL ||
+		qos_pref == NULL) {
+		LOG_DBG("NULL value(s) supplied)");
+		return -EINVAL;
+	}
+
+	ascs = ascs_get(conn);
+	ep = stream->ep;
+
+	if (ep == NULL) {
```

Comment:
`stream->ep` shall always be NULL in this case. If the stream is already configured to a different endpoint, it should be released first. Suggest to return error if `stream->ep` is not NULL